en/docs/extended_topics/model_mixed_parallel.md

```diff
@@ -321,5 +321,5 @@ The first parameter in  `scope.placement` is to specify `cpu` or `gpu`. The seco
 
 Flow parallelism can allow user to specify device for each op. It is very useful for user who master the distributed training to **optimize deeply**.
 
-In addition, OneFlow also provides `oneflow.unpack`, `oneflow.pack`. Combine those with the characteristics of task scheduling in OneFlow. It will make the flow parallelism easier to use and more efficient. We will introduce these in other article.
+In addition, OneFlow also provides `oneflow.unpack`, `oneflow.pack`. Combine those with the features of task scheduling in OneFlow. It will make the flow parallelism easier to use and more efficient. We will introduce these in other article.
```

Comment:
Combine those with the features of task scheduling in OneFlow. It will make the flow parallelism easier to use and more efficient. We will introduce these in other article.
->
Combined with the own features of task scheduling in OneFlow, they will make the flow parallelism easier to use and more efficient. We will introduce these in other article.